config/schedule.rb

```diff
@@ -1,24 +1,15 @@
 set :output, "#{ path }/log/daemons.log"
 
-# REVIEW: get times from Settings
-#  require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
-#  require 'rubygems'
-#  every ENV['synch_every_minutes'].to_i.minutes do
+if :rails_env == 'production'
 
-every 3.minutes do
-  rake 'joegattnet:three_minutes'
-end
-
-# every '*/10 * * * *' do
-#   rake 'joegattnet:ten_minutes'
-# end
+  every 3.minutes do
+    rake 'joegattnet:three_minutes'
+  end
 
-every '0 * * * *' do
-  rake 'joegattnet:one_hour'
-end
+  every '0 * * * *' do
+    rake 'joegattnet:one_hour'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.